Update 65-snapshots.md (#12425)
sveltejs/kit: documentation/docs/30-advanced/65-snapshots.md

Ephemeral DOM state — like scroll positions on sidebars, the content of `<input>` elements and so on — is discarded when you navigate from one page to another.

> For example, if the user fills out a form, but before submitting:
> - clicks a link, then hits the browser's back button
> - use the browser refresh
> 
> The values they filled in will be lost. In cases where it's valuable to preserve that input, you can take a _snapshot_ of DOM state, which can then be restored if the user navigates back.

Reviewer: ```suggestion
For example, if the user fills out a form but navigates away and then back before submitting, or if the user refreshes the page, the values they filled in will be lost. In cases where it's valuable to preserve that input, you can take a _snapshot_ of DOM state, which can then be restored if the user navigates back.
```

I'd like to avoid bullet points here if possible, as they seem awkward. Maybe something like this?
